### ISSUE TITLE
What should I do to preprocess the image from [0-255] to [0.0-1.0] when post training optimization model?

### ISSUE THREAD

**Issue by xRay2016** [openvinotoolkit/open_model_zoo#1969]
I am doing quantization for my own model. Because I train it with pytorch, its input should range from [0,0-1.0] when using the accuracy-cheker. 

But I can't find suitable preprocess method to change the image from [0-255] to [0.0-1.0].

Anyone can help me ?

**Comment by eaidova** [comment]
@xRay2016 in Pytorch changing pixel range is a part of ToTensor operation (it is pretty simple, just divide input data on 255), while in other frameworks it is a part of normalization, so I do not see the problem to perform data scaling with normalization
this preprocessing described in accuracy checker docs:
* normalization - changing the range of pixel intensity values.
  `mean` values which will be subtracted from image channels. You can specify one value for all channels or list of comma separated channel-wise values.
  `std` specifies values, on which pixels will be divided. You can specify one value for all channels or list of comma separated channel-wise values. These parameters support work with precomputed values of frequently used datasets (e.g. cifar10 or imagenet).

in config it would looks like:
```yaml
  preprocessing:
    - type: normalization
       std: 255
```

Another way to embed scaling inside IR using  Model Optimizer (--scale 255 or if you have other normalization mean and scale, you need to multiply them on 255), as example you can look on any torchvision models conversion (e.g. [resnet-50-pytorch](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/resnest-50-pytorch/model.yml#L60-L61)). After that you can use [0, 255] range for input.

**Comment by xRay2016** [comment]
@eaidova OK, I understand how to do that, thank you very much!

**Comment by eaidova** [comment]
@xRay2016 do you have any additional questions or can we close this issue?
